plasmapy/formulary/lengths.py

```diff
@@ -193,16 +202,18 @@ def gyroradius(
     >>> gyroradius(5*u.uG, particle='alpha', T=1*u.eV)
     <Quantity 288002.38... m>
     >>> gyroradius(400*u.G, particle='Fe+++', Vperp=1e7*u.m/u.s)
-    <Quantity 48.23129... m>
+    <Quantity 48.25815... m>
     >>> gyroradius(B=0.01*u.T, particle='e-', T=1e6*u.K)
     <Quantity 0.003130... m>
     >>> gyroradius(0.01*u.T, 'e-', Vperp=1e6*u.m/u.s)
     <Quantity 0.000568... m>
     >>> gyroradius(0.2*u.T, 'e-', T=1e5*u.K)
-    <Quantity 4.94949...e-05 m>
+    <Quantity 4.94957...e-05 m>
```

Comment:
Huh, this changed more than I thought it would!  Though I guess electrons really are going fast at $10^5$ K... 🤔

Reply:
I re ran the command to make sure and yeah it is that big of a change, hopefully I didn't make a logic error

The lorentz factor was calculated to be: 1.0000168641271152126
and the gyroradius was:  4.949576847389539e-05 m